showthread.php

```diff
@@ -482,6 +482,10 @@
 			$showresults = 1;
 		}
 
+		if ($forumpermissions['canvotepolls'] != 1) {
```

Comment:
Mixed code style here - below this section you use 1.8 style (`{` on new line).

Reply:
Good catch. Man, it is so hard to write PHP like that for me now...